doc/source/usage.rst

```diff
@@ -38,47 +38,47 @@ Deploy services
 On the testbed, the services can currently be deployed manually. In the future, these manual
 steps will be automated by Zuul CI.
 
-* Basic infrastructure services (MariaDB, RabbitMQ, Redis, ...)
+* Ceph services
 
   .. code-block:: console
 
-     /opt/configuration/scripts/002-infrastructure-services-basic.sh
+     /opt/configuration/scripts/deploy/002-ceph-services.sh
```

Comment:
Maybe we can renumber the scripts when we rename them anyway to keep them ordered and have enough space for additions. This one would then become e.g. 100-ceph-services.sh ...